README.md

```diff
@@ -16,6 +16,13 @@ Run composer install
 docker run --interactive --rm --volume ${PWD}:/app itkdev/php8.3-fpm:latest composer install
 ```
 
+Working with JS/css...
+
+```shell
+docker compose run --rm php npm install
```

Comment:
Node container?

Reply:
There isnt any, so I figured I'd fix it in a later PR :) Not sure it has to do with the actual update.